samples/utils.js

```diff
@@ -66,10 +70,12 @@ window.chartColors = {
 			var dfactor = Math.pow(10, decimals) || 0;
 			var data = [];
 			var i, value;
+			var rand = cfg.random ? cfg.random(min, max) : this.randF(min, max);
+			var rand01 = cfg.random01 ? cfg.random01() : this.randF();
```

Comment:
random01 does not appear anywhere else, where does it come from, is it a typo or unused?

Reply:
there are two random number generators, one for generating the data and one for generating missing values by chance = 0..1 in some cases one just wanna change one of them but not the other. see violin example